api/proto/info/info.proto

```diff
@@ -507,6 +509,40 @@ message ZInfoNetworkInstance {
   repeated ErrorInfo networkErr = 40;
 }
 
+message UsageInfo {
+  google.protobuf.Timestamp createTime = 1;
+  uint32 refCount = 2;
+  google.protobuf.Timestamp lastRefcountChangeTime = 3; // Last time when refCount changed
+}
+
+message VolumeResources {
+  uint64 maxSizeBytes = 1; // From config
+  uint64 curSizeBytes = 2; // Current disk usage
+}
+
+message ZInfoVolume {
+  string uuid = 1;
```

Comment:
Semantics question: so the way I read this -- these UUIDs are expected to be produced by EVE and simply reported back to the cloud as-is, correct? EVE is in full control of lifecycle of these, right? @eriknordmark @bharani-zededa ?

Reply:
No, the UUIDs will come from the controller.

There is NO (ZERO, ZIPPO) support in this info API to report the current implicitly created volumes.
Based on my understanding and our discussion those will be replaced by explicitly created volumes (with a UUID allocated bt the controller)